jest/private/jest_test.bzl

```diff
@@ -42,13 +48,21 @@ def _impl(ctx):
         ctx.file.config.short_path,
         "--testSequencer",
         relative_file(ctx.file.sequencer.short_path, ctx.file.config.short_path),
+        "--reporters",
```

Comment:
suppose we should also add a `configure_jest_junit_reporter` attribute that default to True but allow it to be disabled incase the user wants to configure something custom instead